File: 프로그래머스/LV3/숫자 게임.py
from collections import deque


def solution(A, B):
    answer = 0
    A.sort()
    B.sort()
    q = deque(B)
    index = 0
    while q:
        b = q.popleft()
        if A[index] < b:
            answer += 1
            index += 1

    return answer

File: 프로그래머스/javascript/LV3/숫자게임.js
function solution(A, B) {
  let answer = 0;
  A.sort((a, b) => a - b);
  B.sort((a, b) => a - b);

  let flag = 0;
  let pa = 0;
  while (flag < B.length) {
    if (A[pa] < B[flag]) {
      answer++;
      pa++;
    }
    flag++;
  }
  return answer;
}
